Helper/Order.php

```diff
@@ -1040,11 +1040,21 @@ public function getExistingOrder($orderIncrementId)
     protected function quoteAfterChange($quote)
     {
         $quote->setUpdatedAt($this->date->gmtDate());
+        // If it's PPC quote make it temporary active
+        // for third party plugins work
+        $is_quote_active = $quote->getIsActive();
```

Comment:
We should use camelCase for variables, but lets leave it like this for now and merge it.